modules/installation-guide/partials/proc_upgrading-che-using-the-cli-management-tool-in-restricted-environment.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
```

Comment:
Is this procedure specific to Openshift 3.11? 
Because on 4.x, the default installer (`chectl --installer`) is `olm` and there the `chectl server:update` wouldn't require the `--che-operator-image` parameter, as it will just be checking the installPlan status and approving it if necessary.
Maybe @tolusha can shed some more light to this, as he is much more "fluent" in this olm/installation stuff.

Reply:
This procedure fits OCP 3.11 as well OCP 4.x if `--installer operator` is used

Reply:
Yes Radim, it's for OCP 3.11. The question is if we should add a NOTE about this fact there, and also think about possible use with OCP 4

@davidfestal @nickboldt  @tolusha  WDYT?

Reply:
So, I would say it also fits k8s as well but I didn't test that.
So I suggest to mention this procedure when `--installer operator` is used and update notes where `OpenShift` is mentioned.

Reply:
How (and where) will this documentation land in downstream docs ? According to what @tolusha said I think that for upstream we can:
-  Explain that this upgrade method is relevant *only* when the existing installation to upgrade has been installed with the `--installer operator` method
- Remind, for clarity, that the `--installer operator` is the default on OCP 3.11, can be used on Kubernetes, and in NOT the default nor recommened on OpenShift 4.x (where OLM / OperatorHub is the recommended way).